nyc.config.js

```diff
@@ -0,0 +1,10 @@
+const config = require('./jest.config');
+
+const removeRootDirectory = pathArray => {
+  return pathArray.map(path => path.slice(10));
+};
+
+module.exports = {
+  include: [...removeRootDirectory(config.collectCoverageFrom), 'pages/**/*.js'],
+  exclude: [...removeRootDirectory(config.coveragePathIgnorePatterns), 'pages/api/__coverage__.js'],
+};
```

Comment:
Wonderful!

Reply:
What's `'pages/api/__coverage__.js'`?

Reply:
That file is required to implement an endpoint to fetch the server-side code coverage. A more detailed explanation can be found [here](https://github.com/bahmutov/next-and-cypress-example).